index.bs

```diff
@@ -5830,6 +5830,74 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 :: None.
 
 
+## Large blob storage extension (<dfn>largeBlob</dfn>) ## {#sctn-large-blob-extension}
+
+This [=client extension|client=] [=authentication extension=] allows a [=[RP]=] to store opaque data associated with a credential. Since [=authenticators=] can only store small amounts of data, and most [=[RPS]=] are online services that can store arbitrary amounts of state for a user, this is only useful in specific cases. For example, the [=[RP]=] might wish to issue certificates rather than run a centralised authentication service.
+
+Note: [=[RPS]=] can assume that the opaque data will be compressed when being written to a space-limited device and so need not compress it themselves.
+
+Since a certificate system needs to sign over the public key of the credential, and that public key is only available after creation, this extension is not defined in the [=registration extension|registration=] context.
+
+Since certificates are sizable relative to the storage capabilities of typical authenticators, user agents SHOULD consider what indications and confirmations are suitable to best guide the user in allocating this limited resource and prevent abuse.
+
+: Extension identifier
+:: `largeBlob`
+
+: Operation applicability
+:: [=authentication extension|Authentication=]
+
+: Client extension input
+::  <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        ArrayBuffer largeBlobWrite;
+        boolean largeBlobRead;
+    };
```

Comment:
These should be wrapped in a single member `largeBlob` of a new dictionary type (see also #1430).

Reply:
We haven't been following that  as noted! I guess we can start… Done.